packages/locale/en.yml

```diff
@@ -1,5 +1,6 @@
 navbar:
   join: Join
+  join.tooltip: Registrations paused. More information coming soon!
```

Comment:
I think we have type definitions for this file. Let's check it before merging

Reply:
Marco I don't see type definitions for this 🤔 Are you sure?

We have a d.ts module declaration for locales:

```ts
declare module '@packages/locale/pt.yml';
declare module '@packages/locale/en.yml';
```

Am I missing something else I should do? If so can you please point it out? Thank you 🙇🏼

Reply:
Double checked, as far as I can see we're ok here, let me KWYT, ty

Reply:
Hmmm sounds good, I just remembered that this typing is Garbage and it's my fault 😅, sorry about that

Reply:
No worries, it's usage is simple, not suffering for missing types there, LGTM